# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libglade10/libglade-10.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./glade/glade-sax.c.orig	Mon Jun 18 12:36:11 2001
+++ ./glade/glade-sax.c	Mon Jun 18 12:36:21 2001
@@ -20,6 +20,8 @@
  * Boston, MA  02111-1307, USA.
  */
 
+typedef char CHAR;
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
